src/resources/views/base/inc/sidebar.blade.php

```diff
@@ -58,11 +58,11 @@ function() { return $(this).attr('href') === full_url; }
       // If not found, look for the link that starts with the url
       if(!$curentPageLink.length > 0){
           $curentPageLink = $navLinks.filter( function() {
-            if ($(this).attr('href').startsWith(full_url)) {
+            if ($(this).attr('href').split('?')[0] === full_url) {
```

Comment:
JavaScript's split always returns an array with at least one value, so even if there's no `?` this still works as expected.